File: cycle-2/print-domains.js
#!/usr/bin/env node

const fs = require("fs")

const script = fs.readFileSync("out/dga.js").toString();

eval(script);

generateDomains()
  .then(domains => {
    console.log(domains.join("\n"));
  })
  .catch(e => console.error(e));

File: cycle-2/print-domains.py
#!/usr/bin/env python3

import importlib, sys

dga_file = sys.argv[1]
dga_mod_name = dga_file.replace("/", ".").replace("\\", ".")

dga_mod = importlib.import_module(dga_mod_name, package = "dga")

print("\n".join(dga_mod.generate_domains()))